server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/user/UserConverter.java

```diff
@@ -15,21 +16,26 @@ public class UserConverter extends BaseGitServiceEntityConverter<org.kohsuke.git
     protected static final Logger logger = LoggerFactory.getLogger(UserConverter.class);
 
     @Override
-    public User convert(@NonNull org.kohsuke.github.GHUser source) {
+    public User convert(@NonNull GHUser source) {
         User user = new User();
-        convertBaseFields(source, user);
         user.setLogin(source.getLogin());
         user.setUrl(source.getHtmlUrl().toString());
```

Comment:
Same here, why are those not in update?

Reply:
You are right, it's actually possible to change your Github login. I assumed it to be immutable initially.